crankshaft-engine/src/service/runner/backend/docker.rs

```diff
@@ -430,11 +436,13 @@ impl crate::Backend for Backend {
                             .await?,
                     );
 
+                    let started = started.take();
+
                     select! {
                         _ = token.cancelled() => {
```

Comment:
Looking at these a bit closer now, does it make sense to add `biased;` here? We always want the cancellation future to take precedence over the other future, right? Perhaps that should be the case for all such `select!`s?

Reply:
I think it makes sense to add `biased;` here; I'll add it.